### DISCUSSION TITLE
Github student account registration

### DISCUSSION THREAD

**Discussion by ShaKk0722** [community/community#113625] [GitHub Education]
### Hi everyone,

When submitting my student card to register for a GitHub student account, how do I know if I was successful? When I press the "Process my application" button, the page reloads, does that mean it was successful?

**Comment by ZuhairTarif** [comment]
Yes it takes time like 48 hours.

**Reply by ShaKk0722** [reply]
It's been 48 hours, I still haven't received a response email. What do I need to do next?

**Comment by samus-aran** [comment]
Hi @ShaKk0722 

We can see that your submission hasn't been submitted. Please check the status of your application [here](https://education.github.com/discount_requests/pack_application). If your application is still pending review then it will appear at the top of the sidebar to the right of the page. If your application has already been approved then you will be redirected to your GitHub Education benefits.

When you try and re-submit we suggest you use a live capture of your evidence.

**Reply by ShaKk0722** [reply]
Thanks for supporting! @samus-aran

**Reply by SuLafice** [reply]
if it says pending, how long will it take to get approved?

**Comment by SebastienXia** [comment]
@ShaKk0722 Have you used the upload a file option for your application ?

**Reply by ShaKk0722** [reply]
Thanks for supporting! @SebastienXia
